neo/d3xp/Game_local.cpp

```diff
@@ -302,6 +302,21 @@ void idGameLocal::Clear( void ) {
 #endif
 }
 
+static DebuggerArgs_t * userDebuggerArgs;
+static bool ( *checkDebuggerBreakpointFnPtr )( void ) = NULL;
+bool IsGameDebuggerActive( idInterpreter *interpreter, idProgram *program, int instructionPointer ) {
+	bool ret = false;
+
+	if ( interpreter != nullptr && program != nullptr ) 	{
+		userDebuggerArgs->interpreter = interpreter;
+		userDebuggerArgs->program = program;
+		userDebuggerArgs->instructionPointer = instructionPointer;
+		ret = checkDebuggerBreakpointFnPtr ? checkDebuggerBreakpointFnPtr( ) : false;
```

Comment:
i thought this is what that user arg was for? 
Will make this arguments of the function itself.

About the naming, i dont want to another function to expose through this mechanism, so ill make naming consistent.

Reply:
user arg was mostly meant for an object that belongs to the engine and that the implementation (*in the engine*) of the exposed function uses - it should be opaque to the game dll, like "this is just another parameter I need to pass to the function, I don't know or care what it is".
in this particular case, you just wouldn't use a user arg at all, because you're only calling global functions anyway

but "normal" args that really come from the game.dll should be passed as regular function arguments